Src/IntegrationTests/TestMvcApp/Controllers/CacheController.cs

```diff
@@ -15,6 +15,33 @@ public CacheController(ICache cache)
             this._cache = cache;
         }
 
+        public IActionResult Get()
+        {
+            const string key = "Get";
+            const string value = "72c41aad-7fec-4603-8f6c-be8a1335b477";
+            this._cache.Remember(key, () => value, TimeSpan.FromDays(1));
```

Comment:
As per the comment in Test file: I would make this endpoint **only** call `Get` and not set anything. Then we can make the tests ensure that the cached values persist through HTTP requests also.

This would apply to `Get` `GetAsync` (which should be removed as per other comment) and `Has`.